### ISSUE TITLE
Microphone

### ISSUE THREAD

**Issue by maxkusnetsov** [MaximumQuiet/ryzentosh#1]
With AppleALC sound is working properly, but input isn't working.

**Comment by ghost** [comment]
It sucks, I see voodoo HDA it's working, but it's not good.

**Comment by maxkusnetsov** [comment]
> It sucks, I see voodoo HDA it's working, but it's not good.

Yeah, I use voodoo when I need a microphone, glad I need it not often
You have probably seen [this guide](https://dortania.github.io/OpenCore-Desktop-Guide/post-install/audio.html#miscellaneous-issues), it's written there we need patches for micro. I'll check AMD-OSX discord for it maybe

**Comment by speri203** [comment]
AppleALC has better sound quality than Voodoo but the only way to support audio in is with a USB dongle. If your headphones come with a detachable cable then you can also get a USB to audio jack which is what I use.

**Comment by maxkusnetsov** [comment]
> AppleALC has better sound quality than Voodoo but the only way to support audio in is with a USB dongle. If your headphones come with a detachable cable then you can also get a USB to audio jack which is what I use.

Yeah, I use bt headphones for now
